src/.vuepress/navbar.ts

```diff
@@ -2,8 +2,8 @@ import { navbar } from "vuepress-theme-hope";
 
 export default navbar([
   "/",
-  { text: "User Guide", icon: "user", link: "courses/tir_user_guide/" },
-  { text: "Admin Guide", icon: "admin", link: "courses/tir_admin_guide/" },
-  { text: "Deployment Guide", icon: "keyboard", link: "courses/tir_deployment_guide/" },
-  { text: "Training Videos", icon: "video", link: "courses/tir_training_videos/" }
+  { text: "User Guide", icon: "iconfont icon-profile", link: "courses/tir_user_guide/" },
```

Comment:
from the docs: https://theme-hope.vuejs.press/guide/interface/icon.html#usage

> If you use this plugin for commercial project documentation, iconfont is strongly NOT recommended as iconfont itself is a study/share platform for designers and developers.

> Every icon is uploaded by users, and you must get authorized from the author for commercial usage. Also, there could be chance where the uploader obeys usage term. And upload icons where its copyright is at 3rd party.

do we want to consider the other supported icon fonts?